tables/isolate_mutations.csv

```diff
@@ -12281,3 +12281,51 @@ SARS-CoV-2/human/DNK/7022055836/2022,N,33,del,NULL,NULL
 SARS-CoV-2/human/DNK/7022055836/2022,N,203,K,NULL,NULL
 SARS-CoV-2/human/DNK/7022055836/2022,N,204,R,NULL,NULL
 SARS-CoV-2/human/DNK/7022055836/2022,N,413,R,NULL,NULL
+B.1.617.2 Spike:+142D+417N,S,19,R,NULL,NULL
+B.1.617.2 Spike:+142D+417N,S,142,D,NULL,NULL

```

Comment:
@KaimingTao too.

Can we get rid of the leading plus symbol? i.e. B.1.617.2 Spike:142D+417N. This should apply for all previous isolates.